xbmc/guilib/GUIIncludes.h

```diff
@@ -11,9 +11,8 @@
 #include "interfaces/info/InfoBool.h"
 
 #include <map>
-#include <set>
 #include <string>
-#include <utility>
+#include <unordered_map>
 #include <vector>
```

Comment:
Adding back the `<utility>` header for `std::pair` should make Jenkins happy.

Reply:
Ahh, looks like you also have to `#include <tinyxml.h>`. And below remove the forward declaration for `TiXmlElement`.